lib/index.js

```diff
@@ -6,9 +6,6 @@ var Client = require('./client');
 var Tunnel = require('./tunnel');
 
 
-var routerInstances = {};
```

Comment:
I'm not sure it's a good idea to drop this feature.

Reply:
what is use case when you initialize multiple tabex instances on the page? 
another way is to listen `destroy` event and remove the cache when a router is destroyed